src-tauri/src/farmer.rs

```diff
@@ -169,7 +169,8 @@ async fn farm(base_directory: PathBuf, farm_args: FarmingArgs) -> Result<(), any
             enable_dsn_archiving: matches!(archiving, ArchivingFrom::Dsn),
             enable_dsn_sync: dsn_sync,
             enable_farming: true,
-            relay_server_node,
+            listen_on,
```

Comment:
I believe you just need to pass empty vector here. We don't care at which interface each plot will listen

Reply:
will do